templates/blog/post_detail.html

```diff
@@ -0,0 +1,24 @@
+{% extends "base.html" %}
+
+{% load crispy_forms_filters %}
+
+{% block content %}
+  <h1>{{ post.title }} (by {{ post.owner.first_name }} {{ post.owner.last_name }})</h1>
+  <h2>{{ post.created_time }}</h2>
+  <p>{{ post.content }}</p>
+  {% if post.comments.all %}
```

Comment:
it's better to use `if post.comments.exists` because your solution fetches all objects and then django checks if it isn't an empty collection.
With .exists it will check the first object and return True or False